File: Phyton/Vetor/soma_vetor.py
n = int(input("Quantos numeros você vai digitar ?"))
vet = [0 for x in range(n)]

for i in range(n):
    vet[i] = float(input("Digite um numero: "))

print("VALORES =", end=" ")
soma = 0
for i in range(n):
    soma += vet[i]
    print(vet[i], end="  ")

print()
print(f"SOMA = {soma}")
print(f"MEDIA = {soma/n:.2f}")

File: C/Vetor/SomaVetor.c
#include<stdio.h>
int main(){

    int n, i;
    double media, soma=0;

        printf("Quantos numeros voce vai digitar?");
        scanf("%d", &n);

            double vet[n];

                for(i=0; i<n; i++){
                    printf("Digite um numero:");
                    scanf("%lf", &vet[i]);
                        soma = soma + vet[i];
                }
        printf("\nValores = ");

            for(i=0; i<n; i++){
                printf("%.2lf ", vet[i]);
            }

        printf("\nSOMA = %.2lf\n", soma);
        media= soma/n;
        printf("MEDIA = %.2lf", media);

return 0;
}
